### PR TITLE
Option to add Subtitle for each date in the view

### PR DESCRIPTION
**Pull request by skymansandy**
Hi Rafal,

We've communicated before regarding the "**unavailable dates**" marking issue and I was unable to do it, I changed my implementation of `equals()` to match that of your examples and it worked. First of all cheers for that.

I actually liked your library very much and while playing with it, I came up with this idea of adding subtitle for each date and wrote down the code for it, We can see the effect in "_Example 2_" of your examples. If it's worth merging, Could you accept my contribution for the library?

Thanks :)

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (RafalManka) [inline .idea/codeStyles/Project.xml:1]: .idea folder should not be committed

File: build.gradle [hunk 2]
```diff
@@ -6,7 +6,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.3'
+        classpath 'com.android.tools.build:gradle:3.3.2'
```
Comment (RafalManka) [inline build.gradle:9]: this should be a part of a different commit

File: examples/build.gradle [hunk 3]
```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 27
-    buildToolsVersion "27.0.3"
+    compileSdkVersion 28
```
Comment (RafalManka) [inline examples/build.gradle:4]: this should be part of a separate commit

File: examples/build.gradle [hunk 4]
```diff
@@ -21,14 +21,14 @@ android {
 }
 
 dependencies {
-    implementation 'com.android.support:design:27.1.1'
-    compile fileTree(include: ['*.jar'], dir: 'libs')
-    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
+    implementation 'com.android.support:design:28.0.0'
```
Comment (RafalManka) [inline examples/build.gradle:24]: this should be part of a separate commit

File: gradle/wrapper/gradle-wrapper.properties [hunk 5]
```diff
@@ -1,6 +1,6 @@
-#Sun Sep 10 18:21:13 GST 2017
+#Tue Mar 26 17:02:58 IST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
\ No newline at end of file
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip
```
Comment (RafalManka) [inline gradle/wrapper/gradle-wrapper.properties:6]: this should be part of a separate commit

File: scrollcalendar/build.gradle [hunk 6]
```diff
@@ -11,12 +11,12 @@ publish {
 }
 
 android {
-    compileSdkVersion 27
-    buildToolsVersion "27.0.3"
+    compileSdkVersion 28
```
Comment (RafalManka) [inline scrollcalendar/build.gradle:14]: this should be part of a separate commit

**Review by RafalManka — CHANGES_REQUESTED** [review]
Great work @skymansandy, thank you for your contribution. Let's clean up the PR first and then we'll look at how to implement a nice clean interface for the widget. I'll sit on it later today too.

**Comment by skymansandy** [comment]
Thank you so much for the suggestions. Will close this PR and create a new PR with changes. :)
